modules/dnn/test/test_halide_layers.cpp

```diff
@@ -1,988 +0,0 @@
-// This file is part of OpenCV project.
```

Comment:
Despite the name, this file should not be removed completely because there are single layer tests for other backends. I suggest renaming.

Reply:
Need further discussion.